server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -136,7 +136,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ML_PACKAGE_LOADER_PLATFORM_ADDED = def(8_512_00_0);
     public static final TransportVersion PLUGIN_DESCRIPTOR_OPTIONAL_CLASSNAME = def(8_513_00_0);
     public static final TransportVersion UNIVERSAL_PROFILING_LICENSE_ADDED = def(8_514_00_0);
-
+    public static final TransportVersion NEED_SHARDS_STATS_ADDED = def(8_515_00_0);
```

Comment:
naming nit

```suggestion
    public static final TransportVersion INCLUDE_SHARDS_STATS_ADDED = def(8_515_00_0);
```

Reply:
It's replaced in 9d20edae06d5d32b9c1678df56465babc6c071e2